src/OpenColorIO/PathUtils.cpp

```diff
@@ -36,7 +34,8 @@ namespace
 // https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?redirectedfrom=MSDN&view=vs-2019
 // "The inode, and therefore st_ino, has no meaning in the FAT, HPFS, or NTFS file systems."
 
-std::string ComputeHash(const std::string & filename)
```

Comment:
Instead of removing `ComputeHash`, only the implementation of `ComputeHash` could handle using an optionally provided function or the default algorithm. Callers of `ComputeHash` should not know or care about that implementation being the default or a provided function, but you could provide the `DefaultComputeHash` function in addition as a means of guaranteeing the standard behavior when a caller needs it.

Reply:
Yeah, you're right. I was thinking that `ComputeHash` was an entry point, but I see it's only `GetFastFileHash`.

Reply:
So, should I let it be `DefaultComputeHash` or make the function available as `ComputeHash`?

Reply:
Keep `DefaultComputeHash`. Disregard my request.